README.md

```diff
@@ -4,30 +4,43 @@
   </a>
 </p>
 <p align="center">
-    <a href="https://datapane.com">Datapane Teams</a> |
-    <a href="https://docs.datapane.com">Documentation</a> |
-    <a href="https://datapane.github.io/datapane/">API Docs</a> |
-    <a href="https://docs.datapane.com/changelog">Changelog</a> |
-    <a href="https://twitter.com/datapaneapp">Twitter</a> |
-    <a href="https://blog.datapane.com">Blog</a>
-    <br /><br />
-    <a href="https://pypi.org/project/datapane/">
-        <img src="https://img.shields.io/pypi/dm/datapane?label=pip%20downloads" alt="Pip Downloads" />
-    </a>
-    <a href="https://pypi.org/project/datapane/">
-        <img src="https://img.shields.io/pypi/v/datapane?color=blue" alt="Latest release" />
-    </a>
-    <a href="https://anaconda.org/conda-forge/datapane">
-        <img alt="Conda (channel only)" src="https://img.shields.io/conda/vn/conda-forge/datapane">
-    </a>
+  <a href="https://datapane.com">Datapane.com</a> |
```

Comment:
When I visit a repo my eyes start looking for “Examples” or “Demo” – shall we link directly to our showcase here?